build.properties

```diff
@@ -0,0 +1,2 @@
+phpcs.excluded = src/Resources/contao/languages
```

Comment:
This file MUST NOT be committed to repositories, it serves as local modification point for the developer.
Please add it to .gitignore and move the lines to `build.default.properties`

Reply:
Ah, now I got the concept. :) Fixed with 207c925ecc7029bb94a53bb2443b0d09b517d489.